ajax/getFormQuestionActorsDropdownValue.php

```diff
@@ -52,7 +52,13 @@
     ]);
 
     // Check if the user can view at least one question
-    if (array_reduce($questions, fn($acc, $question) => $acc || $question->canViewItem(), false) === false) {
+    if (
+        array_reduce(
+            $questions,
+            fn($acc, $question) => $acc || Question::getById($question['id'])->canViewItem(),
```

Comment:
`canViewItem()` may not be enough in many contexts, as it does only specific checks on the item without checking that the user can globally access the items of the corresponding class. For instance, it may return `true` even if the `Question::canView()` returns `false`.

This method should only be used internally by some specific methods, like `CommonDBTM::can()`.

```suggestion
            fn($acc, $question) => $acc || (new Question())->can($question['id'], READ),
```